CMakeLists.txt

```diff
@@ -112,22 +112,33 @@ set(CMAKE_STAGING_PREFIX ${CMAKE_BINARY_DIR}/Root)
 set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR}/Root)
 set(CMAKE_INSTALL_DATAROOTDIR ${CMAKE_BINARY_DIR}/Root/res)
 
-set(GCC_VERSION 11.1.0)
-
-if (${CMAKE_HOST_SYSTEM_NAME} MATCHES SerenityOS)
-    message("Good job on building cmake!")
+if (${CMAKE_CROSSCOMPILING})
+    set(GCC_VERSION 11.1.0)
+    set(LIBRARY_LIBC LibC)
+    set(LIBRARY_LIBCRYPT LibCrypt)
+    set(LIBRARY_LIBM LibM)
+    set(LIBRARY_LIBPTHREAD LibPthread)
+    set(LIBRARY_LIBSYSTEM LibSystem)
+
+    if (${CMAKE_HOST_SYSTEM_NAME} MATCHES SerenityOS)
```

Comment:
When building serenity in Serenity, we won't be cross compiling, right? So this check is in the wrong place.